README.md

```diff
@@ -3,21 +3,20 @@
 [![Version](https://img.shields.io/nuget/v/Terminal.Gui.svg)](https://www.nuget.org/packages/Terminal.Gui)
```

Comment:
Why is this change in this PR? This is already in master. You need to rebase.

Reply:
Rebase issue, sorry.